spec/integration/rack_spec.rb

```diff
@@ -105,6 +105,14 @@
 
       specify { expect(last_response.status).to eql 200 }
       specify { expect(last_response.body).to eq batch_compiled_json }
+      specify {
+        expect(last_response.headers).to eq({
+          "Content-Type"   => "application/json",
+          "ETag"           => "44eaafebcdd94ddac4c4f07ecf0cc80f",
+          "Last-Modified"  => nil,
```

Comment:
Yeah, thinking about it it would be good to do a test on this as we _are_ doing an operation on the dates... (extracting the most recent date).

Sorry, i'd add the test.